### ISSUE TITLE
subscriptions.md (v2 and v3) authentication of user

### ISSUE THREAD

**Issue by coolacid** [justintv/Twitch-API#318]
The documentation should probably spell out that the OAUTH token must be from the channel you are attempting to get access to. 

I assume this is the case as getting subscription data as a non-channel person could be bad :smiling_imp:

**Comment by night** [comment]
It is kind of implied since it states the requirements to make the call:

> *__Authenticated__*, required scope: `channel_subscriptions`

**Comment by coolacid** [comment]
Yep, Kinda implied. But I can authenticate as myself, but not access someone else's subscription stream.  Just suggesting to make it more obvious.  :)
